src/render.rs

```diff
@@ -40,7 +40,7 @@ impl<'a> MarkdownRenderer<'a> {
                 "language-scss",
                 "language-sql",
                 "language-toml",
-                "yaml",
+                "language-yaml",
```

Comment:
My local dev environment isn't set up for working on the backend, so I haven't tested this, but I think this is what it should be.

Reply:
this change is unrelated to "Add TOML syntax highlighting", right?

Reply:
Correct. I saw this while I was looking at the TOML stuff and I figured I would bundle them into one PR.

Reply:
Before this change, YAML highlighting wasn't working:

![image](https://user-images.githubusercontent.com/37223377/93720398-8e779580-fb3d-11ea-9205-6e6ac8bb6c1f.png)

And indeed the `language-yaml` class was missing.